### ISSUE TITLE
login

### ISSUE THREAD

**Issue by MASSI22** [unl1kely/ifa-tcf-catcher#2]
makach win dir login email w mot de pass en moins dir input win neqder ndakhl email w mot de pass

**Comment by unl1kely** [comment]
It works with cookies, not email and password.
Once you login with your own account in your browser -like Chrome- : 

1. Inspect Element
2. Open the Network tab
3. Refresh the page with F5 or Ctrl+R
4. When new requests appear, go to the first one and select it
5. Go to Requests Headers
6. Copy your cookies and pass them into the main python file -in the cookies variable-

**Comment by MASSI22** [comment]
Okay ill give a try
Tell me please have you triéd it before did you succèd and booked
appointement ?
How much succès rate

**Comment by unl1kely** [comment]
This script refreshes the IFA Calendar page each 0.1 seconds to check if there is the available RDV's
You can change it in the main file > last lines of the `main` function > `sleep(0.1)`
The number between the round brackets is the interval between each refresh -request-

About success rate : 
[Success Rate Issue](https://github.com/unl1kely/ifa-tcf-catcher/issues/3)

**Comment by MASSI22** [comment]
THANKS anyway for your work i'll try it
so once i login to my account in chrome i directly go to inspect and
refresh page then copy cookies i don't need go to exams

**Comment by unl1kely** [comment]
Yes Exactly, and there are more instructions and further explanations in the script.

**Comment by MASSI22** [comment]
okay i read it carefully i hope you can improve it with time good bless you
agma
